src/hotspot/cpu/x86/foreign_globals_x86.cpp

```diff
@@ -1,118 +0,0 @@
-/*
```

Comment:
This file was split into foreign_globals_x86_64.cpp, and foreign_globals_x86_32.cpp. The existing implementation was moved to the former.